modules/thermo-cycler/production/uf2conv.py

```diff
@@ -172,7 +172,7 @@ def get_drives():
     if sys.platform == "win32":
         r = subprocess.check_output(["wmic", "PATH", "Win32_LogicalDisk",
                                      "get", "DeviceID,", "VolumeName,",
-                                     "FileSystem,", "DriveType"])
+                                     "FileSystem,", "DriveType"]).decode()
```

Comment:
Like 99% sure this is fine but there's a chance this will fail if someone has a drive with a volume name that isn't ascii (for instance, hanzi) and crash the script, depending on what decode decides is the system locale and what the volume label is encoded as. Might want to give it a try. If it's a problem, you can probably just change the subsequent code to operate on bytes rather than string.

Reply:
Tested by making a volume label on Windows to 'Win(你好)' and no error was thrown while parsing drives. The letters were decoded as 'Win(??)'.
There's no decoding on Mac; `os.listdir(rootpath):` (line 188) returns the chinese charaters of the volume label correctly with no errors or ??

Reply:
👍